### PR TITLE
README: update listing support of ESP8266

### PR DESCRIPTION
**Pull request by emmanuelsearch**
### Contribution description

This PR updates the main README to list support of ESP8266.

### Testing procedure

Not applicable.

### Issues/PRs references

Not applicable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ but not limited to:
 * CoAP
 * CCN-Lite
 * Sigfox
-* LoRaWAN (Experimental)
+* LoRaWAN
```
Comment (miri64) [inline README.md]: Unrelated. Please split at least into a separate commit.

Comment (aabadie) [reply README.md]: +1 and I think we should still consider it as experimental.

**Comment by emmanuelsearch** [comment]
@aabadie @miri64 fixed ;)
